client/layout/guided-tours/docs/API.md

```diff
@@ -0,0 +1,217 @@
+</Step>
+```
+
+## ButtonRow
+
+ButtonRow is a React component to display button controls in Step and takes care of their proper styling. Usually used as the last child of Step to contain all available controls.
+
+### Example
+
+```jsx
+<Step>
+  <p>ButtonRow Example</p>
+  <ButtonRow>
+    <Next step="next-step" />
+    <Quit />
+  </ButtonRow>
+</Step>
+```
+
+## Continue
+
+Continue is a React component that you can use in Step to programmatically continue the tour to other step based on user interaction with Calypso.
+
+### Props
+
+* `step`: (string) Name of the step the tour will advance to.
+* `target`: (string, optional) Name of `[data-tip-target]` that would be watched.
+* `click`: (bool, optional) If true, `onClick` will be listened on `target` DOM node.
+* `hidden`: (bool, optional) If true, this will not render anything in Step, while functionality remains.
+* `icon`: (string, optional) Name of Gridicon to show in custom message.
+* `when`: (function, optional) Redux selector. Once it evaluates to true, tour will advance to `step`.
+* `children`: (nodes, optional) If you provide children, the default content will be overridden.
+
+### Visual Options
+
+There are three ways to define the content of this component.
+
+- default text is "Click to continue." Use just as `<Continue … />` with no children.
+- default text + Gridicon ("Click *icon* to continue.") - provide name of Gridicon as an `icon` prop. Example: `<Continue icon="my-sites" … />`
+- custom content - provide it as children: `<Continue …>My Content</Continue>`
+
+### Functionality Options
+
+There are currently two ways to declare the condition to continue the tour with Continue.
+
+- Binding an `onClick` listener to the step's target DOM node and waiting for the node to be clicked in order to advance the tour.
+- Providing a Redux selector and waiting for it to evaluate to true in order to advance the tour.
+
+```jsx
+// continue when user clicks DOM element with html attribute `data-tip-target="my-sites"`
+<Continue step="next-step" click target="my-sites" />
+
+// continue when Redux selector evaluates to true (in this case after the user opens a preview)
+<Continue step="next-step" when={ isPreviewShowing } />
+```
+
+## Next
+
+Link is a React component that shows a button that allows users to advance tour to another step. To be used inside of `<ButtonRow>`.
+
+### Props
+
+* `step`: (string) Name of the step the tour will advance to.
+
+### Label
+
+Default label is "Next". To override, place your label as a child.
+
+### Example
+
+```jsx
+// with default label
+<Next step="next-step" />
+
+// or with a custom one
+<Next step="next-step">{ translate( 'Custom Label' ) }</Next>`
+```
+
+## Quit
+
+Quit is a React component that shows a button that allows users to quit current tour. To be used inside of `<ButtonRow>`.
+
+### Props
+
+* `primary` (bool, optional) If true, button will be rendered as primary. Use only if Quit is the only available action in that step.
```

Comment:
While we're here, let's add Leif's `subtle` [prop](https://github.com/Automattic/wp-calypso/pull/10035/files#diff-0a062af39e275c16c745f103e205c65e) from #10035.